thavalon-webapp/src/components/Navbar.scss

```diff
@@ -0,0 +1,110 @@
+#navbarContainer {
+    background-color: #333;
+    overflow: hidden;
+
+    #rightContainer {
+        float: right;
+
+        @media screen and (max-width: 600px) {
+            float: none;
+            width: 100%;
+        }
+    }
+
+    .navbarItemLeft {
+        display: block;
+        color: #F2F2F2;
+        text-align: center;
+        padding: 14px 16px;
+        text-decoration: none;
+        font-size: 17px;
+    
+        &:hover {
+            background-color: #DDD;
+            color: #000;
+        }
+    
+        &:active {
+            background-color: #4CAF50;
+            color: #FFF;
+        }
+    
+        &.active {
+            background-color: #4CAF50;
```

Comment:
Could some of these reused color codes go in variables?

Reply:
Will do for most common ones.